README.md

```diff
@@ -220,11 +220,11 @@ var index = await client.GetIndexAsync("movies");
 #### Add or Update Documents <!-- omit in toc -->
 
 ```c#
-var updateStatus = await index.AddDocumentsAsync(new Movie[] { new Movie { Id = "1", Title = "Carol" } } );
-var updateStatus = await index.UpdateDocumentsAsync(new Movie[] { new Movie { Id = "1", Title = "Carol" } } );
+var task = await index.AddDocumentsAsync(new Movie[] { new Movie { Id = "1", Title = "Carol" } } );
```

Comment:
I did not know this convention, and it makes sense!
Could you open an issue so that we can fix it in the future? Definitely a good first issue for the community 😄

Reply:
Created https://github.com/meilisearch/meilisearch-dotnet/issues/216